ariaNotify-polyfill.js

```diff
@@ -176,6 +176,7 @@ if (!("ariaNotify" in Element.prototype)) {
     connectedCallback() {
       this.ariaLive = "polite";
       this.ariaAtomic = "true";
+      this.style.margin = "-1px";
```

Comment:
Consider explicitly offsetting `width` and `height`:
```suggestion
      this.style.marginLeft = "-1px";
      this.style.marginTop = "-1px";
```